### PR TITLE
feat : navigation

### PR DESCRIPTION
**Pull request by sudipkundu999**
File structure:
components>input>input.css => contains CSS for input only
docs>component-presentation>input-presentation.css => contains CSS for presenting the input on the documentation site

### CODE REVIEW

File: Components/navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    width: 100%;
+    height: 5rem;
+    display: flex;
+    align-items: center;
+    padding: 0 2rem;
+}
+
+.cui-nav-brand{
+    display: flex;
+    align-items: center;
```
Comment (dev-prabhat) [inline Components/navigation/navigation.css:11]: you can put display: flex code in a single class and use it anywhere else

**Review by dev-prabhat — CHANGES_REQUESTED** [review]

File: Components/navigation/navigation.css (hunk 1 above)
Comment (sudipkundu999) [reply Components/navigation/navigation.css:11]: Yes, I will surely do that when I add utility classes during refactoring.
For now, I'm keeping it like this.

File: Docs/component-presentation/navigation-presentation.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.cui-navigation-container{
+    border: 1px solid black;
+}
+
+pre code{
+    margin-top: 2rem;
+}
+
```
Comment (GarvG) [inline Docs/component-presentation/navigation-presentation.css:8]: Spacing can be removed

**Review by GarvG — CHANGES_REQUESTED** [review]

File: Components/navigation/navigation.css (hunk 1 above)
Comment (dev-prabhat) [reply Components/navigation/navigation.css:11]: Okay then you are good to go

**Review by dev-prabhat — APPROVED** [review]

File: Docs/component-presentation/navigation-presentation.css (hunk 2 above)
Comment (sudipkundu999) [reply Docs/component-presentation/navigation-presentation.css:8]: this is for readability purposes only

**Review by GarvG — APPROVED** [review]

File: Components/navigation/navigation.css (hunk 1 above)
Comment (GarvG) [reply Components/navigation/navigation.css:11]: okay , looks good to merge

**Pull request merged**